collector/envoy/docker-compose.override.yaml

```diff
@@ -15,7 +15,7 @@ services:
   loadgen:
     depends_on: [nginx_proxy]
     image: williamyeh/hey
-    command: ["-z", "10m", "-c", "50", "-q", "50", "http://front-envoy:8080"]
+    command: ["-z", "10m", "-c", "50", "-q", "50", "http://localhost:8080/service/1"]
```

Comment:
This was useful at the time for moving the metrics on envoy, but I don't think we have to keep this loadgen shim.